easyaccess/easyaccess.py

```diff
@@ -4,7 +4,8 @@
 from __future__ import absolute_import
 
 __author__ = 'Matias Carrasco Kind'
-__version__ = '1.2.1b'
+from easyaccess.version import __version__
```

Comment:
These changes make it impossible to call `easyaccess.py` directly or through the symbolic link `bin/easyaccess`. Either attempt yields the error:
```
Traceback (most recent call last):
  File "./easyaccess.py", line 7, in <module>
    from easyaccess.version import __version__
  File "/Users/alex/easyaccess.py", line 7, in <module>
    from easyaccess.version import __version__
ImportError: No module named version
```
I believe that this is happening because it is trying to import the `version` module from the `easyaccess.py` file instead of the package directory. (With the symbolic link, there must be some absolute path dereferencing leading to the same problem.) 

The reason that things work when the package is installed with `setup.py`, is that the source code for the executable is copied from `easyaccess.py` to `easyaccess` and thus tricks python into importing from the package which is in the PYTHONPATH rather than the local copy of the file itself.

As an example, if you were to change the name of this file to `tmp_easyaccess.py`, then it would be possible to execute it directly (make sure to remove any `.pyc` files). So basically, the problem is that there are too many things named `easyaccess` and python is getting confused which one to look in for the submodules. One solution would be to move the source code into `__init__.py`, another would be to rename `easyaccess.py` (for example, to `easy_or` or 'connect').